Library/Homebrew/utils/github.rb

```diff
@@ -501,6 +501,24 @@ def check_for_duplicate_pull_requests(name, tap_remote_repo, state:, file:, args
     end
   end
 
+  def get_long_build_pull_requests_files(name, repo)
+    pull_requests = search_issues("org:#{name}", repo: repo, state: "open", label: "\"long build\"")
+    prs = pull_requests.map { |pull_request| pull_request["number"] }
+    files = []
+    prs.each do |pr|
+      pr_files = API.open_rest(url_to("repos", name, repo, "pulls", pr, "files"))
```

Comment:
This rest query appears twice (once here, and once in `get_pr_files`). Is there some way to recycle these query results? In particular I think you query the files in a `long build` PR at least twice via the REST API.

Reply:
I added a `get_pr_files`, that can now be used at 3 places in the file.

What do you mean by "query the files twice"?

Reply:
By "query the files twice" I mean making a REST API call to ask GitHub about the files modified in a pull request.

I actually suspect this will get rate-limited pretty quickly if we make one (and possibly more) API call per PR, since we get only a few thousand of these API calls in a given time period, and there are over a hundred open PRs at any given time.